dodola/core.py

```diff
@@ -28,15 +35,26 @@ def bias_correct_bcsd(
         variable name used in training data
     out_variable : str
         variable name used in downscaled output
+    method : str
+        method to be used in the applied bias correction
     ds_predicted : Dataset
         bias corrected future model data
     """
-
-    # note that time_grouper='daily_nasa-nex' is what runs the
-    # NASA-NEX version of daily BCSD
-    # TO-DO: switch to NASA-NEX version once tests pass
-    model = PointWiseDownscaler(BcsdTemperature(return_anoms=False))
-    model.fit(gcm_training_ds[train_variable], obs_training_ds[train_variable])
-    predicted = model.predict(gcm_predict_ds[train_variable]).load()
+    if method == "BCSD":
+        # note that time_grouper='daily_nasa-nex' is what runs the
+        # NASA-NEX version of daily BCSD
+        # TO-DO: switch to NASA-NEX version once tests pass
+        model = PointWiseDownscaler(BcsdTemperature(return_anoms=False))
+        model.fit(gcm_training_ds[train_variable], obs_training_ds[train_variable])
+        predicted = model.predict(gcm_predict_ds[train_variable]).load()
+    elif method == "QDM":
+        # instantiates a grouper class that groups by day of the year
+        # centered window: +/-15 day group
+        group = sdba.Grouper("time.dayofyear", window=31)
+        model = sdba.adjustment.QuantileDeltaMapping(group=group, kind="+")
+        model.train(gcm_training_ds[train_variable], obs_training_ds[train_variable])
+        predicted = model.adjust(gcm_predict_ds[train_variable])
+    else:
+        raise ValueError("this method is not yet supported")
```

Comment:
Minor/optional thing:

This message feels funny to me because the ValueError might get raised on a misspelling or some crazy method that we'd never support. Suggest  something like`"this method is not supported"`. It removes the promise of future work for whatever input just triggered the exception.

Reply:
That's fair, updated per your suggestion.